docs/testing/fixture.md

```diff
@@ -0,0 +1,21 @@
+# Fixture
+
+## Dependencies teardown
+
+When using dependency injection in tests, it's important to properly tear down resources after tests complete.
+
+Without proper teardown, the Python event loop might close before resources have a chance to shut down properly, leading to errors like `RuntimeError: Event loop is closed`.
+
+You can set up automatic teardown using a pytest fixture:
+
+```python
+import pytest_asyncio
+from typing import AsyncGenerator
+
+from my_project import DIContainer
+
+@pytest_asyncio.fixture(autouse=True)
+async def di_container_teardown() -> AsyncGenerator[None]:
+    yield
+    await DIContainer.tear_down()
```

Comment:
it's better to wrap yield with try ... finally to be sure that tear down will always be called

Reply:
The try/finally block in this case is actually redundant because pytest's fixture handling already ensures that teardown code (after the yield) is always executed, even if the test fails.

Reply:
maybe, but it's a good habit to always wrap)